### PR TITLE
Fix #11902 issue

### LINKED ISSUE #11902: "Record Attempts to Alter Logon and Logout Events" issues on Ubuntu 22.04
#### Description of problem:

2 rules using "CIS Ubuntu 22.04 Level 2 Server Benchmark" fails in the "Record Attempts to Alter Logon and Logout Events" section:

- Record Attempts to Alter Logon and Logout Events - faillog (xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillog)
- Record Attempts to Alter Logon and Logout Events - tallylog (xccdf_org.ssgproject.content_rule_audit_rules_login_events_tallylog)

They look for /var/log/faillog and /var/log/tallylog files, but none of these were mentioned in the "CIS Ubuntu Linux 22.04 LTS Benchmark" guide (see 6.3.3.12 Ensure login and logout events are collected - Page 816):

```
Remediation:
Edit or create a file in the /etc/audit/rules.d/ directory, ending in .rules extension,
with the relevant rules to monitor login and logout events.
Example:
# printf "
-w /var/log/lastlog -p wa -k logins
-w /var/run/faillock -p wa -k logins
" >> /etc/audit/rules.d/50-login.rules
Merge and load the rules into active configuration:
# augenrules --load
```

#### SCAP Security Guide Version:

master branch

#### Operating System Version:

Ubuntu 22.04 LTS

#### Steps to Reproduce:

1. Edit file /etc/audit/rules.d/50-login.rules to include:
-w /var/log/lastlog -p wa -k logins
-w /var/run/faillock -p wa -k logins
2. Execute augenrules --load
3. Execute:

oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis_level2_server --rule xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillog ssg-ubuntu2204-ds.xml

oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis_level2_server --rule xccdf_org.ssgproject.content_rule_audit_rules_login_events_tallylog /root/scap-security-guide/build/ssg-ubuntu2204-ds.xml

#### Actual Results:

```
root@ubuntu:~# oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis_level2_server --rule xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillog /root/scap-security-guide/build/ssg-ubuntu2204-ds.xml
W: oscap: File ssg-ubuntu2204-cpe-oval.xml has already been registered in Source DataStream session: /root/scap-security-guide/build/ssg-ubuntu2204-ds.xml
Title   Record Attempts to Alter Logon and Logout Events - faillog
Rule    xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillog
Result  fail

root@ubuntu:~# oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_cis_level2_server --rule xccdf_org.ssgproject.content_rule_audit_rules_login_events_tallylog /root/scap-security-guide/build/ssg-ubuntu2204-ds.xml
W: oscap: File ssg-ubuntu2204-cpe-oval.xml has already been registered in Source DataStream session: /root/scap-security-guide/build/ssg-ubuntu2204-ds.xml
Title   Record Attempts to Alter Logon and Logout Events - tallylog
Rule    xccdf_org.ssgproject.content_rule_audit_rules_login_events_tallylog
Result  fail
```

#### Expected Results:

`Result  pass`

On Ubuntu 22.04 pam_tally2 was replaced by pam_faillock and the CIS guide refer to pam_faillock.

Why "/var/log/faillog" is also checked but not mentioned in the CIS guide? 

Who should be updated? The "CIS Ubuntu Linux 22.04
LTS Benchmark" guide or the "SCAP Security Guide" ?

### PR DESCRIPTION
**Pull request by marcofortina**
#### Description:

On Ubuntu 22.04 pam_tally2 was replaced with pam_faillock and checks should be done on files referenced by the CIS guide:

```
-w /var/log/lastlog -p wa -k logins
-w /var/run/faillock -p wa -k logins
```

#### Rationale:

- Fixes #11902

#### Review Hints:

Build the product:

```
./build_product ubuntu2204
```

To test these changes with Ansible:

```
ansible-playbook build/ansible/ubuntu2204-playbook-cis_level2_server.yml --tags "audit_rules_login_events_faillock"
```

To test changes with bash, run the remediation sections: `xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillock`

Expected output:

```
Title   Record Attempts to Alter Logon and Logout Events - faillock
Rule    xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillock
Result  pass
```

### CODE REVIEW

**Review by dodys — CHANGES_REQUESTED** [review]
please add and remove the ids in the rule.yml files 

to help you out, take a look on:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillock/rule.yml#L54
and 
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillog/rule.yml#L40

File: products/ubuntu2204/profiles/cis_level2_workstation.profile [hunk 1]
```diff
@@ -9,6 +9,9 @@ description: |-
 extends: cis_level1_workstation
 
 selections:
+    ### Variables
+    - var_accounts_passwords_pam_faillock_dir=run
```
[the remark this answers is not in the document]
Comment (marcofortina) [reply products/ubuntu2204/profiles/cis_level2_workstation.profile]: done

**Review by marcofortina — COMMENTED** [review]
Code reviewed

**Review by dodys — APPROVED** [review]
lgtm, thanks!

**Pull request merged**